### ISSUE TITLE
{} placed at end of /etc/datadog-agent/datadog.yaml

### ISSUE THREAD

**Issue by rtizzy** [DataDog/ansible-datadog#93]
Using the latest version of the Ansible galaxy role from the Galaxy repo, I have the following issue. 

Replication:

Set the API variable
Enabled dd agent 6 (Believe this same error may happen regardless of agent 6)
Run the role. 

Result:

The service will fail to start and a {} is placed at the end of the datadog.yaml file. 
Removing the {} causes the service to start properly.

**Comment by rtizzy** [comment]
I believe this is unintended behavior causes when no extra config values are being set. 

If I do something like set a tag via variables, everything works as expected. 

This works:

```
datadog_api_key: REDACTED
# Testing the new beta agent due to superior performance and functionality
datadog_agent6: true

datadog_config:
  tags: "customer:{{ customer }}"

customer: bobsdogstore
```

This doesn't work. 

```
datadog_api_key: REDACTED
# Testing the new beta agent due to superior performance and functionality
datadog_agent6: true
```

Looking at main.yml you see the following variable set. 

```
# default datadog.conf options
datadog_config: {}
```

This behavior will need to be modified at some level to work properly when config is not set. 

The best manner of handling this may be to simply unset the datadog_config default variable. 

Please comment.

**Comment by k4nar** [comment]
I'm also encountering this issue with the version 2.0.1. It worked fined prior to 2.0 & with the agent v5.

**Comment by Meovel** [comment]
Out of this topic, just wanna mention there's a typo in README
> Variables:
> datadog_agent5 - install an agent5 instead of agent5 (default to false)

**Comment by sonjz** [comment]
same.

![image](https://user-images.githubusercontent.com/1714763/37164569-b0846e78-22c8-11e8-86a1-b276e1385aae.png)

temporary fix:

```
# just added a default value from datadog.yaml.example so my service will start!
- name: Setup Datadog
  include_role:
    name: Datadog.datadog
  vars:
    datadog_api_key: "{{ datadog.api_key }}"
    datadog_config:
      skip_ssl_validation: no  # by default this is set
```

**Comment by pbudzon** [comment]
Updated to the new version of the role and monitoring blew up because of this bug... not very nice :(

Thanks for the workaround @sonjz

**Comment by ohaiwalt** [comment]
Any word on when this will release? Ran into the issue yesterday :(

**Comment by hush-hush** [comment]
@rtizzy Thanks for reporting the issue and sorry for the late delay. I'll get on it right away.

**Comment by hush-hush** [comment]
The fix is available in `2.0.2`.
